### PR TITLE
Add support for Tencent Cloud host alias (VM ID)

### PR DESCRIPTION
**Pull request by hush-hush**
### What does this PR do?

Add support for Tencent Cloud host alias (VM ID).

To avois triggering security alerts on AWS, GCP and other cloud provider when the agent query an unknown IP we disabled Tencent cloud provider by default

### CODE REVIEW

File: pkg/metadata/host/host.go [hunk 1]
```diff
@@ -136,6 +138,14 @@ func getHostAliases() []string {
 	} else if k8sAlias != "" {
 		aliases = append(aliases, k8sAlias)
 	}
+
+	tencentAlias, err := tencent.GetHostAlias()
+	if err != nil {
+		log.Debugf("no Tencent Host Alias: %s", err)
+	} else if azureAlias != "" {
```
Comment (ogaca-dd) [inline pkg/metadata/host/host.go:145]: Is `azureAlias` a typo?

**Review by ogaca-dd — COMMENTED** [review]
LGTM, just a potential typo.

File: pkg/util/tencent/tencent.go [hunk 2]
```diff
@@ -32,6 +32,14 @@ func IsRunningOn() bool {
 	return false
 }
 
+// GetHostAlias returns the VM ID from the Azure Metadata api
```
Comment (remeh) [inline pkg/util/tencent/tencent.go:35]: Typo in the comment as well

**Review by truthbk — APPROVED** [review]
Looks good, thanks for those tests! 🙇

**Pull request merged**